extensions/objects/templates/image-editor.component.ts

```diff
@@ -1012,7 +1012,7 @@ export class ImageEditorComponent implements OnInit, OnChanges {
     // especially if there are a lot. Changing the cursor will let the
     // user know that something is happening.
     document.body.style.cursor = 'wait';
-    gifFrames({
+    window.GifFrames({
```

Comment:
Why is this changed? I think the more direct imports makes more sense.

Reply:
This is done to make the file testable. Previously, since `gifFrames` was private and it could not be mocked.

Reply:
Hmm, is this the only possible solution? 

Because adding it to `window` is indirect and can lead to other issues, it can make string typing a bit harder and especially since we are slowly moving to imports instead of requires and these provide some advanced functionality this might make it actually even more complicated to move to import here.

Reply:
Currently using `require` returns a function rather than an object. Hence `spyOn` cannot be used since it requires an object as its first parameter. However, after converting to import if it can be imported as `<object>.gifFrames` then we can remove it from `window`.

Reply:
If `import` would be possible for this library then it would be best I think. If it cannot be used I'm fine with the `window` approach but with a TODO issue to fix this later.

Reply:
I have attempted `import * alias from 'path'`. This should have resulted in `alias` being a wrapper object but it was a function instead. Shall I create an issue and comment it in the files image-editor.component.ts file?

Reply:
@vojtechjelinek #13676 PTAL

Reply:
and who should I assign the issue to?